samples/client/petstore/typescript-angular-v2/default/api/pet.service.ts

```diff
@@ -63,10 +63,10 @@ export class PetService {
     /**
      * 
      * @summary Add a new pet to the store
-     * @param body Pet object that needs to be added to the store
+     * @param pet Pet object that needs to be added to the store
      */
-    public addPet(body: Pet, extraHttpRequestParams?: RequestOptionsArgs): Observable<{}> {
-        return this.addPetWithHttpInfo(body, extraHttpRequestParams)
+    public addPet(pet: Pet, extraHttpRequestParams?: RequestOptionsArgs): Observable<{}> {
```

Comment:
@mnahkies thanks for the PR. Looks like you're using an old version of OpenAPI Generator as the body name in OAS v2 is not preserved. Can you please pull the latest master into your branch and regenerate the samples again?

Reply:
Hmm, seems like this script actually generates to the same directory as the openapi2 script, is that intentional?
```
$ mvn clean package && ./bin/openapi3/typescript-angular-petstore-all.sh
```

Results in no changes (based on latest master)

Running 
```
$ mvn clean package && ./bin/typescript-angular-petstore-all.sh
```

Reverts all changes to samples to current master. I'll push this, but seems like some work needs to be done around the typescript client samples with respect to openapi3